app/components/sign-up-form/styles.scss

```diff
@@ -35,3 +35,25 @@ input,
         width: #{$i * 20%};
     }
 }
+
+& {
+    &__progress-bar-primary-bright {
```

Comment:
I'd use a `@each` loop here since they're all the same. http://thesassway.com/intermediate/if-for-each-while#each Not sure if you can do that with variable lookups, but worth exploring.

Reply:
I can look into that

Reply:
SASS doesn't support variable interpolation in a variable name, but you could do it with a map:

```SASS
$progress-bar-bg-bright: (
    primary: #428bca,
    info: #5bc0de,
    success: #5cb85c,
    warning: #f0ad4e,
    danger: #d9534f,
);

& {
    @each $class, $color in $progress-bar-bg-bright {
        &__progress-bar-#{$class}-bright {
            background-color: $color;
        }
    }
}
```

Reply:
👍 When docker finally restarts I'll give this a try.